api/APIv2.md

```diff
@@ -246,13 +248,13 @@ Request:
 
 The request mime type MUST be "application/x-proto-binary".
 The body MUST be a protobuf message of type AuthContainer where the AuthBody is a protobuf message of type [attest.ZAttestReq](./proto/attest/attest.proto).
-`ZAttestReq` has `reqType` which MUST be filled by EVE to identify type of the request. If `reqType` is `ATTEST_REQ_CERT`, `certs` should be filled with certificates used by EVE. If `isIMutable` attribute is not set, Controller should reject any request that would overwrite an existing certificate previously published. If `reqType` is `ATTEST_REQ_NONCE`, no other field needs to be set in `ZAttestReq`. If `reqType` is `ATTEST_REQ_QUOTE`, then `quote` MUST be populated with the quote generated by Trusted Platform Module.
+`ZAttestReq` has `reqType` which MUST be filled by EVE to identify type of the request. If `reqType` is `ATTEST_REQ_CERT`, `certs` should be filled with certificates used by EVE. If `isIMutable` attribute is not set, Controller should reject any request that would overwrite an existing certificate previously published. If `reqType` is `ATTEST_REQ_NONCE`, no other field needs to be set in `ZAttestReq`. If `reqType` is `ATTEST_REQ_QUOTE`, then `quote` MUST be populated with the following information: `attestData` containing [TPMS_ATTEST structure defined in TCG](https://trustedcomputinggroup.org/wp-content/uploads/TPM-Rev-2.0-Part-2-Structures-01.38.pdf), `signature` containing signature of `TPMS_ATTEST` by the attestation signing key, `pcr_sha256_values` containing values of PCR 0-15, `event_log` containing TPM Event Log (aka measurement logs) with each event entry of the format `TpmEventLogEntry`,  `eve_version` containing current EVE version, `fw_version` containing the current firmware version, `gps_info` containing GPS co-ordinates of device's geo-location.
```

Comment:
Since you are touching this, can you call the device a device instead on an EVE?
Note that "current EVE version" is current, but the other uses of EVE are presumably about a device.

Reply:
Done